practice-two/src/components/common/Card/Card.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Card Component
+ * @param maxW - The maximum width of the card. Defaults to 'xl'.
+ * @param variant - The variant of the card style. Defaults to 'primary'.
+ * @param children - The content to be displayed inside the Card component.
+ * @param onChange - The callback function .
+ * @returns {JSX.Element} - The Card component.
+ */
+const Card = ({
+  maxW = 'xl',
+  variant = 'primary',
+  children,
+  onChange = () => {},
```

Comment:
is card have onChange event? I don't think it's correct in this case because u don't have any change in card